doc/basic_configuration.rst

```diff
@@ -17,10 +17,17 @@ configuration to generate the necessary Jobs on the Jenkins master.
 Create / fork configuration repository
 --------------------------------------
 
-First you need to either fork the
+First you need to either fork or clone the
 `ros_buildfarm_config <https://github.com/ros-infrastructure/ros_buildfarm_config>`_
 repository or create a repository containing the same configuration files.
 
+**Important:**
+Since ``ros_buildfarm_config`` files need to be accessed from within docker images,
+you **can not** use a local file system path (``file:/``) to reference them.
```

Comment:
```suggestion
you **can not** use a local file system path (`file://`) to reference them.
```

Reply:
Thanks @gavanderhoorn 
I always thought the second dash was a idiosyncrasy of http and not required for ``file``.
Well, in practice it isn't required, but it sill seems to be the usual way of doing it:
https://en.wikipedia.org/wiki/File_URI_scheme